js-callback-functions/js-callback-functions_user-login-callback-2-with-argument/js/index.js

```diff
@@ -5,3 +5,50 @@ function handleUserLogin(onSuccess) {
 }
 
 // The exercise starts here!
+
+
+
+//#Task 1:
+function showWelcomeMessage(){
+  console.log('"Welcome Jane Doe! You are logged in now."')
+}
+
+handleUserLogin(showWelcomeMessage);
+
+//#Task 2:
+handleUserLogin(showWelcomeMessage, () => {
+
```

Comment:
i think the task was to write the handleUserLogin also as an arrow function. So inside the second function (which should have no arguments) (inside the arrow) there should be the same console log message.